lib/safira_web/router.ex

```diff
@@ -28,6 +28,7 @@ defmodule SafiraWeb.Router do
 
       get "/user", AuthController, :user
       get "/attendee", AuthController, :attendee
+      get "/leaderboard", LeaderboardController, :index
```

Comment:
I really dont like a route with no table associated, it feels kinda weird

Reply:
We don't need a table since this is sorting and filtering the attendees